js/littlesis.js

```diff
@@ -387,8 +387,10 @@ var littlesis = (function() {
 		  	  templates: {
 		  	    notFound: '<div class="entity-not-found">No results found. Try searching again; maybe you misspelled something? <div class="new-entity-footer"><button class="new-entity-btn primary">CREATE NEW ENTITY</button></div></div>',
 		  	    suggestion: function(data) {
+              var urlSlug = '/entities/' + data.id + '-' + data.name.replace(/\s/g, '_');
```

Comment:
it's not 100% necessary as ` /entities ` is a valid url, but does this ` data ` object contain the primary extension? If it does, it'd be nice to construct urls with `/org` or `/person` instead of `/entities `